entry_point.py

```diff
@@ -59,6 +61,66 @@ def change_values(file_name, getter_func):
         print(new_str.replace('\n', ''))
 
 
+def get_varname(var, prefix="ODOORC_"):
+    """
+    Cleans the string and return the variable ready to use in the config file
+
+    :param var: variable name as comes from the source
+    :param prefix: prefix to be deleted from the string
+    :return: Sting in lowercase without the prefix
+    """
+    r = re.compile("^"+prefix)
+    return r.sub("", var, 1).lower()
+
+
+def get_odoo_vars(getter_func):
+    res = dict()
+    for key, value in getter_func():
+        if key.startswith("ODOORC_"):
+            var = get_varname(key)
+            res.update({var: value.strip()})
+    return res
+
+
+def append_values(file_name, getter_func):
+    """
+    Append values to a config file, new values are gotten from env vars. All the variables must start with ODOORC_
+    otherwise will be ignored (so we make sure not to append all variables)
+
+    :param str file_name: Config file name
+    :param getter_func: Function that will be used for getting new values, this should return a dict
+    in the following format:
+        {
+          "ODOORC_VARIABLE_NAME": 123,
+          "ODOORC_ANOTHER_VARIABLE_NAME": "asd"
+        }
+    """
+    variables = get_odoo_vars(getter_func)
+
+    for line in fileinput.input(file_name, inplace=True):
+        new_str = line
+        logger.debug("Line read: %s", line.strip())
+        parts = line.split("=")
```

Comment:
if `new_str` contains the same as `line` why are you using both?

Reply:
NVM, I see that the value of `line` is the initial value of `new_str` but that can change below, nothing to see here, go on